### PR TITLE
Update quest scripts for 'Summoning the Princess' quest series.

### PR DESCRIPTION
**Pull request by miraco**
Includes https://github.com/cmangos/tbc-db/issues/681

These are the first Quests i started reworked/added scripts based on Video/Sniffs. I didnt want to merge it directly cause i have multiple questions to this.

1. Time Stamps and Delay.
Example
`ServerToClient: SMSG_QUEST_GIVER_QUEST_COMPLETE (0x2A83) Length: 39 ConnIdx: 0 Time: 12/13/2021 18:54:37.079 Number: 21172
ServerToClient: SMSG_UPDATE_OBJECT (0x27C2) Length: 1086 ConnIdx: 1 Time: 12/13/2021 18:54:37.907 Number: 21233
ServerToClient: SMSG_CHAT (0x2BAD) Length: 127 ConnIdx: 0 Time: 12/13/2021 18:54:40.875 Number: 21856
ServerToClient: SMSG_UPDATE_OBJECT (0x27C2) Length: 631 ConnIdx: 1 Time: 12/13/2021 18:54:42.766 Number: 21875
`
These were the events that got used for Quest 652. I used timers 1 second to spawn npc after completing quest, 4secs for saying text and 6 secs to attack start (update_object).  

NPC gets spawned  0.828 seconds after completing quest (according to the sniffs) and says text  2.968 seconds after spawn.
I just rounded it up to 1 seconds and 3 seconds. Since we use milliseconds for delay, we could also use the sniffed values for it.

@killerwife

### CODE REVIEW

File: Updates/0035_summoning_the_princess.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+UPDATE `quest_template` SET `StartScript`=692, `CompleteScript`=692 WHERE `entry`=692;
+UPDATE `quest_template` SET `DetailsEmote1`='0', `DetailsEmote2`='0', `DetailsEmote3`='0', `DetailsEmote4`='0' WHERE (`entry`='692');
+UPDATE `broadcast_text` SET `ChatTypeID`='2' WHERE (`Id`='870');
+
+-- Summoning the Princess
+UPDATE `quest_template` SET `DetailsEmote1`='0', `DetailsEmote2`='0', `DetailsEmote3`='0', `DetailsEmote4`='0' WHERE (`entry`='656');
+
+DELETE FROM dbscripts_on_event WHERE id = 420;
+INSERT INTO `dbscripts_on_event` (`id`, `priority`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `data_flags`, `dataint`, `x`, `y`, `z`, `o`, `comments`) VALUES 
+('420', '0', '0', '10', '2755', '3000000', '0', '0', '0','0', '-931.73', '-3111.81', '48.517', '3.27404', 'Summoning the Princess - Summon Myzrael'),
```
Comment (miraco) [inline Updates/0035_summoning_the_princess.sql:59]: The second questions is about this line.
After using the item  [Scroll of Myzrael]  id 4472, the npc gets spawned. I watched multiple videos and did this quest on retail tbc and it seems this dont get spawned on fixed point, instead it mostly got spawned infront of the Player.
Will need more research for this.

Comment (killerwife) [reply Updates/0035_summoning_the_princess.sql]: This generally indicates use of a spell with unit field created_by_spell

**Comment by killerwife** [comment]
Retail has generally a precision within 100ms of what it actually is. I have a suspicion that their script engine doesnt tick every server update. In most cases i also coalesce it to closest half a second. But i have seen scripts with increments of 1200ms in some cases. These generally imply a spell however so your instinct is fine.

File: Updates/0035_summoning_the_princess.sql (hunk 1 above)
Comment (miraco) [reply Updates/0035_summoning_the_princess.sql]: The Spell (4141) use Id: 61 (SPELL_EFFECT_SEND_EVENT) as Effect0. 
I watched some more videos and i think it was just luck that it spawned directly next to me. 

In this 2 videos she has the same coords as in my personal sniff from retail tbc. 
https://www.youtube.com/watch?v=g9QlmT3hzlw
https://www.youtube.com/watch?v=g9QlmT3hzlw

Comment (killerwife) [reply Updates/0035_summoning_the_princess.sql]: Then give it the same exact coords and dont worry about it. Dont need to overcomplicate things for ourselves.

Comment (miraco) [reply Updates/0035_summoning_the_princess.sql]: Yeah im sorry, just always want to have everything perfect :x

**Pull request merged**